modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/BaseLayout/index.jsx

```diff
@@ -20,27 +30,94 @@ const Layout = () => {
 		}
 	}, [accountKey]);
 
-	return (
-		<div className="d-flex position-relative w-100">
-			<div>
-				<div className="align-items-center cp-layout-header d-flex justify-content-between ml-4 mt-4">
-					<ProjectBreadcrumb />
-				</div>
+	const {client, featureFlags} = useAppPropertiesContext();
 
-				{hasSideMenu && <SideMenu />}
-			</div>
+	const {
+		data: myUserAccount = {accountBriefs: [], organizationBriefs: []},
+		isLoading: loadingMyUserAccount,
+	} = useSWR({key: '/projects'}, getMyUserAccount);
+
+	const myFLSOrganizationBriefIds = useMemo(
+		() =>
+			myUserAccount?.organizationBriefs
+				?.filter(({name}) => name.includes('FLS'))
+				.map(({id}) => id),
+		[myUserAccount?.organizationBriefs]
+	);
+
+	const {data: organizations = [], isLoading: loadingOrganizations} = useSWR(
+		{
+			key: '/organizations',
+			organizationIds: myFLSOrganizationBriefIds,
+		},
+		myUserAccount ? getFLSOrganizationsAccounts(client) : null
+	);
+
+	const organizationProjectsERC = organizations.map(
+		({externalReferenceCode}) => externalReferenceCode
+	);
+	const isFLSOrganization = organizationProjectsERC.some(
+		(externalReferenceCode) => externalReferenceCode === accountKey
+	);
+
+	const teamMembersERC = myUserAccount?.accountBriefs?.map(
+		({externalReferenceCode}) => externalReferenceCode
+	);
+	const isTeamMember = teamMembersERC.some(
+		(externalReferenceCode) => externalReferenceCode === accountKey
+	);
+
+	const {data: myAccount} = useGetMyUserAccount();
+	const isAdminUserAccount = hasAdminUserAccount(myAccount);
+
+	const liferayContactERC =
+		myUserAccount.accountBriefs
+			?.filter(({roleBriefs}) =>
+				roleBriefs.some(
+					(roleBrief) => roleBrief.name === 'Provisioning'
+				)
+			)
+			.map(({externalReferenceCode}) => externalReferenceCode) || [];
+
+	const isliferayContact = liferayContactERC.includes(accountKey);
+
+	const accountPermission =
+		isTeamMember ||
+		isAdminUserAccount ||
+		isFLSOrganization ||
+		isliferayContact;
+
+	if (loadingOrganizations && loadingMyUserAccount) {
+		return <ClayLoadingIndicator />;
+	}
 
-			<div className="d-flex flex-fill pt-4">
-				<div className="mx-4 px-2 w-100">
-					<Outlet
-						context={{
-							setHasSideMenu,
-						}}
-					/>
+	if (accountPermission) {
+		return (
+			<div className="d-flex position-relative w-100">
+				<div>
+					<div className="align-items-center cp-layout-header d-flex justify-content-between ml-4 mt-4">
+						<ProjectBreadcrumb />
+					</div>
+
+					{hasSideMenu && <SideMenu />}
+				</div>
+
+				<div className="d-flex flex-fill pt-4">
+					<div className="mx-4 px-2 w-100">
+						<Outlet
+							context={{
+								setHasSideMenu,
+							}}
+						/>
+					</div>
 				</div>
 			</div>
-		</div>
-	);
+		);
+	}
+
+	if (!accountPermission && featureFlags.includes('ISSD-65')) {
```

Comment:
does this need a feature flag? i didnt see that as a requirement in the ticket

Reply:
@kylembischof , It's not in the ticket requirements, when we implement something new visually, that doesn't interfere with a flow, we put a feature flag with the "epic" ticket for safety, as a good practice. If it's not necessary, can I remove it? I look forward to your reply.

Reply:
okay, i dont think its necessary to put feature flags on every little visual change. i would say this ticket doesn't need a feature flag. 
i think feature flags should be mainly used on larger changes, or if we have to hold a feature that we don't want to release yet. it will save us time not putting feature flags on little things when not necessary so we don't have to go back and remove them all.

Reply:
okay